CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## main / unreleased
 
+* [ENHANCEMENT] Add a `prep_downscale` mutating admission webhook that prepares ingesters for shutdown before downscaling. #47
```

Comment:
Can this be written in a way that doesn't refer to ingesters specifically? I understand that's the use case in mind here, but `rollout-operator` is intentionally generic.

Reply:
I've changed it to pods. That terminology is used in other parts of the changelog.